CHANGELOG.md

```diff
@@ -29,6 +32,8 @@
 
 - Bugfix: Telepresence correctly reports an intercept port conflict instead of panicking with segfault.
 
+- Bugfix: Intercept questions about the cluster ingress were made more intuitive. 
```

Comment:
should this be here? 2.4.4 is already out

Reply:
ha yep deleted